src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -82,8 +82,8 @@ public static class WebApiConfig
 
         private static void AddMessageHandlers(HttpConfiguration config)
         {
-            config.MessageHandlers.Add(new WebScriptHostHandler(config));
             config.MessageHandlers.Add(new SystemTraceHandler(config));
```

Comment:
When debugging with Kranthi, we weren't getting any logs from the host when it was in this state (we were sending ping requests). So I'm moving the trace handler BEFORE the other handler, which does the "DelayUntilHostReady" logic.